website/web/templates/vuln.html

```diff
@@ -393,9 +392,40 @@ <h5>Tags</h5>
         document.getElementById("epss-score").parentNode.parentNode.remove();
       });
     }
-
   })
 
+
+  // Open the tab specified with an anchor in the URL.
+  window.onload = function() {
+    const hash = window.location.hash;
+    const tabLink = document.querySelector(`a[href="${hash}"]`);
+    if (tabLink) {
+      const tab = new bootstrap.Tab(tabLink);
+      tab.show();
+      selected_tab = tabLink.getAttribute("id");
+      switch (selected_tab) {
+        case "comments-tab":
+          loadComments();
+          break;
+        case "bundles-tab":
+          loadBundles();
+          break;
+        case "sightings-tab":
+          loadSightings();
+          break;
+        default:
+          console.log("Not a valid day.");
```

Comment:
Valid day?

Reply:
haha, it comes from previous tests of the switch statement. I just removed this console.log. Now it opens the tag of related vulnerabilities, in case the anchor in not correct.